app/_src/policies/meshmetric.md

```diff
@@ -231,6 +238,29 @@ scrape_configs:
         client_id: "prometheus-two"
 ```
 
+#### OpenTelemetry (experimental)
+
+```yaml
+backends:
+  - type: OpenTelemetry
+    openTelemetry: 
+      endpoint: http://otel-collector.observability.svc:4317
+```
+
+This configuration tells {{site.mesh_product_name}} Dataplane Proxy to push metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
```

Comment:
nit: we use "data plane proxy" (lowercase, separate words)

Reply:
changed